src/design-system/theme/index.tsx

```diff
@@ -23,6 +23,11 @@ export const darkTheme = createTheme({
 });
 
 const ThemeProvider = ({ children, colorScheme }: { children: ReactNode; colorScheme?: "light" | "dark" }) => {
+  useEffect(() => {
+    colorScheme === "dark"
+      ? document.body?.style.setProperty("background", "#0e061c")
+      : document.body?.style.setProperty("background", "fff");
```

Comment:
nit: this will make sure that if we will update the background color in the palette, this will get updated too
```suggestion
    document.body?.style.setProperty("background",  colorScheme === "dark" ? darkTheme.palette.background.default : theme.palette.background.default);
```